cannon/testdata/example/mt-cond/main.go

```diff
@@ -0,0 +1,303 @@
+// Copyright 2011 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Copyright 2009 The Go Authors.
+//
```

Comment:
On second thought, I think this format a little confusing.  Can you update to the following format that includes a permalink to the original source code?:
```suggestion
// Portions of this code are derived from code written by The Go Authors.
// See original source: https://github.com/golang/go/blob/e67c0f0c8f1b49937eb197f6172ae75633767db7/src/sync/cond_test.go
//
// --- Original License Notice ---
//
// Copyright 2009 The Go Authors.
//
```